.gitignore

```diff
@@ -1,5 +1,7 @@
-/.idea
-/.vscode
-/node_modules
+.ipynb_checkpoints
+.idea
+.vscode
+node_modules
 *.DS_Store
 .env
+data-analysis/private_data
```

Comment:
All data that stores publicly identifiable information should be stored on your local in this directory so it won't be checked into GitHub.